main/store/state/index.js

```diff
@@ -440,6 +440,7 @@ const initial = {
       ethereum: {
         1: {
           gas: {
+            fees: {},
```

Comment:
@floating do we need to add these or no? also do we need migrations or will these just be populated on initial load?

Reply:
It depends how the action works and how/when the values are consumed, i.e. will something look for it before it has been added by an action and then fail. 

It's likely best to add a migration.